src/example-types/text.js

```diff
@@ -37,7 +47,7 @@ module.exports = {
       return result
     }
 
-    if (lookAtUntouched) fieldName += '.lowercased'
+    if (lookAtKeyword) fieldName += '.keyword'
```

Comment:
You can probably just leave this as lookAtUntouched so it doesn't affect any other code, and use the field name .untouched instead until we go to .keyword everywhere. Same with the lookAtKeyword above (can be changed back to lookAtUntouched).